src/main/java/mate/jdbc/dao/impl/FlightDaoImpl.java

```diff
@@ -0,0 +1,130 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.FlightDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Flight;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class FlightDaoImpl implements FlightDao {
+    @Override
+    public Flight create(Flight flight) {
+        String query = "INSERT INTO flights(number, carrier, manufacturer_id) VALUES (?, ?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection
+                         .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)
+        ) {
+            statement.setString(1, flight.getNumber());
+            statement.setString(2, flight.getCarrier());
+            statement.setLong(3, flight.getManufacturer());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                flight.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert flight to DB : "
+                    + flight, e);
+        }
+        return flight;
+    }
+
+    @Override
+    public Optional<Flight> get(Long id) {
+        String query = "SELECT * FROM flights WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)
+        ) {
```

Comment:
```suggestion
                 PreparedStatement statement = connection.prepareStatement(query)) {
```

Reply:
Can I left Flight, or change all to Pilote, for example?